data/entities.json5

```diff
@@ -202,6 +202,20 @@
       'img.youtube.com',
       'fcmatch.youtube.com',
     ],
+    products: [
+      {
+        name: 'YouTube Embedded Player',
+        urlPatterns: [
+          '.*\\.youtube\\.com/embed/.*',
```

Comment:
this makes the strongest case yet for finally just giving up on json and moving this file to `entities.js` I think I might do that before publish

Reply:
So this you will move the data over after this PR lands?

Reply:
yep 👍